overlord/ifacestate/ifacemgr.go

```diff
@@ -174,6 +174,10 @@ func (m *InterfaceManager) StartUp() error {
 			return err
 		}
 	}
+	if snapdAppArmorServiceIsDisabled() {
+		s.Warnf(`the snapd.apparmor service is disabled; snap applications will likely not start.
+Run "systemctl enable snapd.apparmor" to correct this.`)
```

Comment:
The service needs to be started, so we should probably suggest to run this:
```suggestion
Run "systemctl enable --now snapd.apparmor" to correct this.`)
```